update import for is_loaded

ahaessly opened the pull request:
add "id_loaded" column to schema
add "partition_id != '__UNPARTITIONED__'" to query so we don't get int conversion error

rsasch reviewed (changes requested):
With the "is_loaded" column added to the schema, I ran into errors trying to ingest data, I think because the number of columns doesn't match up.  Currently, that column is created here: https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsImportGenomes.wdl#L885 so it doesn't need to be in the original schema.

ahaessly commented:
I moved creating the sample_info table to the GvsAssignIds.wdl so the GvsImportGenomes is no longer updating the sample_info table. I removed the add column because this fails on repeated runs of the GvsImportGenomes wdl (it only needs to be added once, so it makes sense to actually do that on table creation)

rsasch reviewed (approved):
Ah, ok — sorry I missed that 👍🏻
